besser/bot/nlp/llm/llm_huggingface.py

```diff
@@ -53,10 +58,16 @@ def set_num_previous_messages(self, num_previous_messages: int) -> None:
     def initialize(self) -> None:
         self.pipe = pipeline("text-generation", model=self.name)
 
-    def predict(self, message: str, parameters: dict = None) -> str:
+    def predict(self, message: str, parameters: dict = None, session: 'Session' = None) -> str:
         if not parameters:
             parameters = self.parameters
-        outputs = self.pipe([{'role': 'user', 'content': message}], return_full_text=False, **parameters)
+        context_messages = []
+        if self._global_context:
+            context_messages.append({'role': 'user', 'content': f"Context: {self._global_context}"})
+        if session and session.id in self._user_context:
+            context_messages.append({'role': 'user', 'content': f"Context: {self._user_context[session.id]}"})
+        messages = context_messages + [{'role': 'user', 'content': message}]
```

Comment:
In Huggingface, the message pattern for an LLM must be user/assistant/user/assistant... Please update this line by:

messages = merge_llm_consecutive_messages(context_messages + [{'role': 'user', 'content': message}])

This will merge all user messages into a single one